engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/rollingformulamulticolumn/RollingFormulaMultiColumnOperator.java

```diff
@@ -33,33 +32,32 @@
 import static io.deephaven.util.QueryConstants.NULL_INT;
 
 public class RollingFormulaMultiColumnOperator extends UpdateByOperator {
-    private static final int BUFFER_INITIAL_CAPACITY = 128;
-
-    @NotNull
-    final TableDefinition tableDef;
-
-    final FormulaColumn formulaColumn;
-    final String[] inputColumnNames;
-    final Class<?>[] inputColumnTypes;
-    final Class<?>[] inputVectorTypes;
-    final Class<?>[] inputComponentTypes;
-
-    protected WritableColumnSource<?> primitiveOutputSource;
-    protected WritableColumnSource<?> outputSource;
-    protected WritableColumnSource<?> maybeInnerSource;
-    ChunkType outputChunkType;
+    private static final int BUFFER_INITIAL_CAPACITY = 512;
+
+    private final String formula;
+    private final TableDefinition tableDef;
+    private final QueryCompilerRequestProcessor compilationProcessor;
```

Comment:
I couldn't follow where the `formula` and `compilationProcessor` are being used.

Reply:
They are used in `RollingFormulaMultiColumnOperator#copy`, but that method is used to support `PartitionedTable` `proxy()` calls and can be made more efficient (by supplying the compiled formula column, eg.).